code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -1,95 +1,98 @@
+		/obj/item/weapon/reagent_containers/food/snacks/grown/wheat = list("flour" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/oat = list("flour" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/cherries = list("cherryjelly" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/bluecherries = list("bluecherryjelly" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/egg = list("eggyolk" = 20),
+
+		//Grinder stuff, but only if dry. Add it to the dried list below.
+		/obj/item/weapon/reagent_containers/food/snacks/grown/coffee = list("coffeepowder" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/coffee/robusta = list("coffeepowder" = 0, "morphine" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/tea = list("teapowder" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/tea/astra = list("teapowder" = 0, "salglu_solution" = 0),
+
+
+		//Juicer Stuff
+		/obj/item/weapon/reagent_containers/food/snacks/grown/corn = list("corn_starch" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/tomato = list("tomatojuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/carrot = list("carrotjuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/berries = list("berryjuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/banana = list("banana" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/potato = list("potato" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/citrus/lemon = list("lemonjuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/citrus/orange = list("orangejuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/citrus/lime = list("limejuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/watermelon = list("watermelonjuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/watermelonslice = list("watermelonjuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/berries/poison = list("poisonberryjuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/pumpkin = list("pumpkinjuice" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/blumpkin = list("blumpkinjuice" = 0),
+
+		//Random Meme-tier stuff!!
+		/obj/item/organ/internal/butt = list("fartium" = 20),
+		/obj/item/weapon/storage/book/bible = list("holywater" = 100)
+	)
+
+	var/list/dried_items = list(
+		//Grinder stuff, but only if dry,
+		/obj/item/weapon/reagent_containers/food/snacks/grown/coffee/robusta = list("coffeepowder" = 0, "morphine" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/coffee = list("coffeepowder" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/tea/astra = list("teapowder" = 0, "salglu_solution" = 0),
+		/obj/item/weapon/reagent_containers/food/snacks/grown/tea = list("teapowder" = 0)
+	)
+
+	var/list/holdingitems = list()
```

Comment:
Please do not initialize lists directly on the object, or in `/new` please use the `LAZY_` list helpers when working with lists.

More about that can be found [here](https://github.com/HippieStation/HippieStation13/blob/master/code/__HELPERS/lists.dm#L459)